tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp3.1</TargetFrameworks>
+    <RuntimeIdentifiers>win-x86;win-x64;linux-x64;osx-x64</RuntimeIdentifiers>
```

Comment:
Probably if target framework has no windows, then the rids won't have win- either.  See line above ...
 <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp3.1</TargetFrameworks>

Reply:
I will make it conditional as well. Now, we target net47 by default and netcoreapp if nix.